SEED-platform/seed
# Paginate sensor readings

#### Any background context you want to provide?

#### What's this PR do?

#### How should this be manually tested?
1. Go to any property
2. Add a datalogger
3. add sensors with `example-sensor-metadata.xlsx`
4. add sensor readings with `example-sensor-readings.csv`
5. Observe that the sensor readings are paginated.

#### What are the relevant tickets?
#3327 

#### Screenshots (if appropriate)
